server/data/licenceTypes.ts

```diff
@@ -1,3 +1,16 @@
+export enum LicenceStage {
+  UNSTARTED = 'UNSTARTED',
+  DEFAULT = 'ELIGIBILITY',
+  ELIGIBILITY = 'ELIGIBILITY',
+  PROCESSING_RO = 'PROCESSING_RO',
+  PROCESSING_CA = 'PROCESSING_CA',
+  APPROVAL = 'APPROVAL',
+  DECIDED = 'DECIDED',
+  MODIFIED = 'MODIFIED',
+  MODIFIED_APPROVAL = 'MODIFIED_APPROVAL',
+  VARY = 'VARY',
+}
+
```

Comment:
moved this down from the service layer, think a lot more stuff should be pulled down into the model